webdriver/webdriver.go

```diff
@@ -9,17 +9,26 @@ import (
 	"os"
 	"runtime"
 	"testing"
+	"time"
 
 	"github.com/phayes/freeport"
 	"github.com/tebeka/selenium"
 )
 
+// Flags
 var (
 	debug            = flag.Bool("debug", false, "Turn on debug logging")
 	browser          = flag.String("browser", "firefox", "Which browser to run the tests with")
 	startFrameBuffer = flag.Bool("frame_buffer", frameBufferDefault(), "Whether to use a frame buffer")
 )
 
+const (
+	// LongTimeout is the timeout for waiting the full page to load, with
+	// data coming from Datastore. You may not need this if you only need
+	// to wait for the initial Polymer rendering.
+	LongTimeout = time.Second * 30
```

Comment:
```suggestion
	longTimeout = time.Second * 30
```
A local constant

Reply:
And change the case in the test if you decide to make this change

Reply:
Although this is only used within this package (i.e. technically no need to be exported), it's used widely in multiple files and there's a convention in this package to export this kind of helpers (see the other helpers in this package).

The main reason for this change was mentioned in the PR description: "to extend the timeout everywhere as we started to see timeouts more often since the Go 1.12 migration (which is presumably caused by the slower Datastore emulator)".